src/shared/components/CreditCardForm/index.tsx

```diff
@@ -48,16 +66,12 @@ const ZuoraPaymentForm: FC = () => {
         setClient(window.Z)
       }
     }
-
-    return () => {
-      client && document.body.removeChild(document.getElementById(scriptId))
-    }
```

Comment:
Why are we removing this?

Reply:
This is causing the functionality to fail if you click on `Change Payment` multiple times. It's unable to find the script in the body but it apparently is getting added and the functionality exists on dom.